### PR TITLE
Feature/add ordering for varisity and campus

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: higher_health/templates/healthcheck_questionnaire.html [hunk 1]
```diff
@@ -136,6 +136,10 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
                   </li>
                 {% endfor %}
               </ul>
+              {% if form.errors %}
+                <br/>
+                <p class="error">Please correct all the errors before submitting.</p>
+              {% endif %}
```
Comment (rudigiesler) [inline higher_health/templates/healthcheck_questionnaire.html:142]: Shouldn't this be with the button, rather than inside the medical `div` with a `br`?

**Review by rudigiesler — COMMENTED** [review]
Looks good, just one small comment

File: higher_health/templates/healthcheck_questionnaire.html (hunk 1 above)
Comment (miltontony) [reply higher_health/templates/healthcheck_questionnaire.html:142]: With the button it appears outside the form, which doesn't look right. 
image

Comment (rudigiesler) [reply higher_health/templates/healthcheck_questionnaire.html:142]: Ah right, we probably want it in the white part. 👍

**Review by rudigiesler — APPROVED** [review]
Looks good, just one small comment

**Pull request merged**